web/app/js/src/jsoneditor-loader.js

```diff
@@ -25,16 +30,19 @@ $(document).on('click', '.jsonLoader', function(){
 });
 
 $(document).on('click', '.jsonSaver', function(){
-  $.post('app/controllers/EntityAjaxController.php', {
-    updateJsonFile: true,
-    id:currentFileUploadID,
-    upload_id: currentFileItemID,
-    type: currentFileType,
-    content: JSON.stringify(editor.get())
-  }).done(function(data){
-    notif(data);
-    if(data.msg==='JSON file updated successfully'){
-      $('.jsonSaver').attr('disabled', 1).text('Saved').css('cursor','default');
-    }
-  });
+  var formData = new FormData();
+  var blob = new Blob([JSON.stringify(editor.get())], { type: 'application/json' });
+  formData.append('replace', true);
+  formData.append('upload_id', currentFileItemID);
+  formData.append('id', currentFileUploadID);
+  formData.append('type', 'experiments');
+  formData.append('csrf', $('meta[name ="csrf-token"]').attr('content'));
```

Comment:
is this really needed, given that [this](https://github.com/elabftw/elabftw/blob/master/web/app/js/src/common.js#L12-L17) exists?

Reply:
or why not use jquery?